frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -52,14 +52,12 @@ action.GitMachete.BaseFastForwardMergeToParentAction.description=Fast-forward me
 action.GitMachete.BaseFastForwardMergeToParentAction.notification.title.ff-fail=Fast-forward merge failed
 action.GitMachete.BaseFastForwardMergeToParentAction.notification.text.ff-fail=Fast-forward merge branch ''{1}'' to ''{0}'' failed.
 action.GitMachete.BaseFastForwardMergeToParentAction.notification.text.ff-success=Branch ''{1}'' has been fast-forward merged to ''{0}''.
-action.GitMachete.BaseFastForwardMergeToParentAction.task-title=Fast-forward merging
-action.GitMachete.BaseFastForwardMergeToParentAction.task-subtitle=Fast-forwarding\u2026
+action.GitMachete.BaseFastForwardMergeToParentAction.task-title=Fast-forwarding\u2026
 
 
 action.GitMachete.BaseSyncToParentByMergeAction.description-action-name=Merge parent into branch
 action.GitMachete.BaseSyncToParentByMergeAction.description=Merge branch ''{0}'' into ''{1}''
 action.GitMachete.BaseSyncToParentByMergeAction.task-title.current=Merging
-action.GitMachete.BaseSyncToParentByMergeAction.task-title.non-current=Checking out & merging
```

Comment:
Why no longer needed? 🤔

Reply:
nope, see frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseSyncToParentByMergeAction.java

Now we depend on git4idea methods and they take care of the tasks and their titles